Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.10.8-slim
+
+WORKDIR /app
+
+COPY app/main.py .
+
+RUN pip install requests
```

Comment:
It's a good practice to include a `requirements.txt` file for Python dependencies and use `RUN pip install -r requirements.txt` instead. This makes it easier to manage dependencies and update them as needed.